modular_skyrat/modules/salon/code/hair_tie.dm

```diff
@@ -74,39 +77,43 @@
 	if(!picked_hairstyle || !actual_hairstyle)
 		return
 	user.visible_message(
-			span_notice("[user.name] takes [src] out of [user.p_their()] hair."),
-			span_notice("You let down your hair!"),
-		)
+		span_notice("[user.name] takes [src] out of [user.p_their()] hair."),
+		span_notice("You let down your hair!"),
+	)
 	user.set_hairstyle(actual_hairstyle, update = TRUE)
 	actual_hairstyle = null
 
 /obj/item/clothing/head/hair_tie/AltClick(mob/living/user)
 	. = ..()
 	if(!(user.get_slot_by_item(src) == ITEM_SLOT_HANDS))
+		balloon_alert(user, "hold in-hand!")
 		return
 	user.visible_message(
-	span_danger("[user.name] puts [src] around [user.p_their()] fingers, beginning to flick it!"),
-	span_notice("You try to flick [src]!"),
+		span_danger("[user.name] puts [src] around [user.p_their()] fingers, beginning to flick it!"),
+		span_notice("You try to flick [src]!"),
 	)
 	flick_hair_tie(user)
 
+///This proc flicks the hair tie out of the player's hand, tripping the target hit for 1 second
 /obj/item/clothing/head/hair_tie/proc/flick_hair_tie(mob/living/user)
-	//find out if this is the syndie hair tie
-	var/syndie = istype(src, /obj/item/clothing/head/hair_tie/syndicate)
-	if(!do_after(user, syndie ? FIRE_SPEED_SYNDIE : FIRE_SPEED_REGULAR, src))
+	if(!do_after(user, fire_speed, src))
 		return
-	//pick and build the right projectile
-	var/obj/projectile/proj_to_fire = syndie ? /obj/projectile/bullet/hair_tie/syndicate : /obj/projectile/bullet/hair_tie
-	var/obj/projectile/bullet/hair_tie/proj = new proj_to_fire (drop_location())
+	//build the projectile
+	var/obj/projectile/bullet/hair_tie/proj = new projectile_to_fire (drop_location())
 	//clone some vars
 	proj.name = name
 	proj.icon_state = icon_state
 	//add projectile_drop
 	proj.AddElement(/datum/element/projectile_drop, type)
-	//fire the projectile
+	//aim
 	proj.firer = user
 	proj.fired_from = user
-	proj.fire(dir2angle(user.dir) + (syndie ? 0 : rand(-30, 30)))
+	//calculate precision
+	var/fire_angle = dir2angle(user.dir)
+	if(!istype(src, /obj/item/clothing/head/hair_tie/syndicate))
```

Comment:
I meant it more like the `30` degrees here could just be a variable on the class, I just tested and `rand(-0, 0)` will always return `0` so that's not a concern to be had. It's just going to be a lot cleaner than this.